app/src/main/java/com/hamdan/forzenbook/view/ForzenbookActivity.kt

```diff
@@ -91,4 +93,30 @@ class ForzenbookActivity : ComponentActivity() {
             }
         }
     }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        if (!isChangingConfigurations) {
```

Comment:
Rebase this PR

Reply:
It was already rebased, I don't know why this one change stayed. It's removed now, I looked through the commit to check for any other things but this was the only thing.